Description: Remove the command that generates gschemas.compiled file.
             This file should not be regenerated while packaging.
             The dropped command is
                 glib-compile-schemas /usr/share/glib-2.0/schemas
             It failed since there are no permissions to put files in
             this directory during building process.
Author: Mònica Ramírez Arceda <monica@debian.org>
Last-Update: 2016-02-26
Index: regexxer-0.10/Makefile.am
===================================================================
--- regexxer-0.10.orig/Makefile.am
+++ regexxer-0.10/Makefile.am
@@ -108,7 +108,7 @@ update_icon_cache   = $(GTK_UPDATE_ICON_
 @INTLTOOL_XML_NOMERGE_RULE@
 
 install-data-hook: install-update-icon-cache
-	$(GLIB_COMPILE_SCHEMAS) $(gsettingsschemadir)
+#	$(GLIB_COMPILE_SCHEMAS) $(gsettingsschemadir)
 #$(gsettingsupdateschemacache) $(gsettingsschemadir)
 
 uninstall-hook: uninstall-update-icon-cache
